core/src/main/java/org/infinispan/topology/ClusterTopologyManagerImpl.java

```diff
@@ -118,12 +118,12 @@ public void inject(Transport transport,
       this.cacheManagerNotifier = cacheManagerNotifier;
       this.cacheManager = cacheManager;
       this.timeService = timeService;
+      this.eventLog = eventLogManager.getEventLogger();
```

Comment:
while you still may want to grab the eventlogmanager here, retrieving the actual eventlogger should still happen in the start() method, since the server event logger is different